request.el

```diff
@@ -550,6 +550,7 @@ and requests.request_ (Python).
 .. _jQuery.ajax: http://api.jquery.com/jQuery.ajax/
 .. _requests.request: http://docs.python-requests.org
 "
+  (declare (indent defun))
```

Comment:
how about adding a defvar and a switch here like `(if request-indent-defun defun nil)`? I'd very much like better indent here too.

Reply:
I'm pretty sure that `declare` forms are not evaluated.

Reply:
How about 
(If request-indent-defun (declare (indent defun))

Reply:
The indent style must be shared with everyone.
If someone can set the indent style in any way, indentation changes will occur many times simply by alternating between those using the traditional indentation style and those using the current master indentation style. This is a nightmare.